### ISSUE TITLE
Size of icons in top panel too big

### ISSUE THREAD

**Issue by matguidi** [material-shell/material-shell#572]
**Describe the bug**
In my Dell XPS 15 7590, with 3840 x 2160 resolution, the "+" icon and the icon on the right of available layouts are too big.

**Expected behavior**
The icons should be smaller.

**Screenshots**
![Schermata del 2021-01-27 15-55-43](https://user-images.githubusercontent.com/59918868/106009039-36870780-60b8-11eb-9824-b51585eecf68.png)

**To Reproduce**
- Occurs on:
  - [X] X.Org
  - [ ] Wayland
- GNOME Shell version: 3.38
- Distro: Manjaro 20.2.1
- Steps to reproduce, if applicable:
1. Load Gnome desktop with Material Shell enabled
2. Bug

**Comment by szelemeh** [comment]
I agree with the issue and this is my suggestion on how to solve the problem:
There is the so-called Zen mode, where you see only one application at once and don't see any of the Material Shell bars. You can enter it by pressing Super+Esc. 
My idea is to add another mode that would make those bars smaller and put only small indicators that _there are_ some other workspaces or apps in the row. That would make use of Super+WASD shortcuts better because the user would see where it can move with these shortcuts but still have more space than it currently has.
Sorry if this comment does not comply with the rules of posting them, but I just wanted to help :hugs:

**Comment by vladgtr** [comment]
Likewise on Huawei Matebook X 2020, with 3000x2000 resolution

**Comment by csskevin** [comment]
Any updates on this?
